eatsmart/templates/inspections/establishment_detail.html

```diff
@@ -6,18 +6,34 @@
 {% block body_id %}establishment-{{ establishment.pk }}{% endblock %}
 
 {% block content %}
+
 <div class="page-header">
   <h1>{{ establishment|title }}</h1>
 </div>
 
 <div class='row'>
-    <div class='col-xs-12 col-sm-6 col-sm-push-6'>
+    <div class='col-xs-12 col-sm-2 col-sm-push-10'>
         <div class='score bg-info'>
             <p class='value'>{{ latest.score }}</p>
             <p class='outof'>{% trans 'out of 100' %}</p>
         </div>
     </div>
-    <div class='col-xs-12 col-sm-6 col-sm-pull-6'>
+    
+    <div class="col-xs-12 col-sm-5 col-sm-push-2">  
```

Comment:
let's wrap this around an if statement, durham data doesn't contain risk factors.